src/components/config/index.ts

```diff
@@ -97,6 +111,11 @@ export const setupConfigFromFile = async (flags: any, watch: boolean) => {
   }
 
   const parsed = parseConfig(path, type)
+  if (flags['create-config'] && (flags.har || flags.postman)) {
+    writeConfigFile(flags, parsed)
+    return
+  }
+
```

Comment:
i think this shouldn't be here. it makes things complicated. check my comment in the setupConfig function.

Reply:
removed